src/envoy/utils/config.cc

```diff
@@ -47,19 +68,14 @@ bool ReadV2Config(const Json::Object &json, Message *message) {
   if (!json.hasObject(kV2Config)) {
     return false;
   }
-  std::string v2_str = json.getObject(kV2Config)->asJsonString();
-  Status status = ParseJsonMessage(v2_str, message);
-  auto &logger = Logger::Registry::getLog(Logger::Id::config);
-  if (status.ok()) {
-    ENVOY_LOG_TO_LOGGER(logger, info, "V2 mixer client config: {}",
-                        message->DebugString());
-    return true;
+  return ReadConfig(json, message, kV2Config);
+}
+
+bool ReadV1Config(const Json::Object &json, Message *message) {
+  if (!json.hasObject(kV1Config)) {
```

Comment:
this checking can be part of ReadConfig() too.

Reply:
Done.